### DISCUSSION TITLE
Where can we buy a server? از کجا سرور بخریم؟

### DISCUSSION THREAD

**Discussion by hiddify-com** [hiddify/Hiddify-Manager#162] [Q&A سوالات رایج] [answered]
از کجا میشه سرور خرید؟

**Comment by hiddify-com** [comment, marked as answer]
از همه جا میتونید 
اینجا ما لیستی از سرورها را جمع آوری کردیم
شما هم به ما کمک کنید لیست را کامل تر کنیم
https://github.com/hiddify/awesome-iran-freedom/blob/main/vps-providers.md

**Comment by majid129** [comment]
از سایت هتزنر خرید کن البته با پاسپورت

**Comment by erfannf** [comment]
برای خرید و راه اندازی سرورهای RELAY در ایران چه پیشنهادی دارید که امن!!! و ارزان باشه دوستان؟

**Comment by aligh98** [comment]
[pq.hosting](https://pq.hosting/en/)
[aeza](https://aeza.net/)
[liteserver](https://liteserver.nl/)
این ها نسبتا ارزان هستن و البته رمزارز هم بعنوان روش پرداخت قبول میکنند
سایت Lowendbox هم سر بزنید برای آفرهای جذاب و مفت. (البته کیفیت سرویس هایی که معرفی میکنند ممکنه مطلوب نباشه)

**Comment by m0000hamad** [comment]
این سایت عالیه پینگ طلایی قیمت هم نسبت به جاهای دیگه خوبه چندساله مشتری هستم.فقط موقع ثبت نام واحد پول رو ریال بزارید:
gigo.host

**Reply by Bahman5931** [reply]
ریاب نداره که 3  ta  گزاشته کدوم ریال هست

**Comment by daryoosh53** [comment]
این سایت فیکه
من یک دامنه خریدم هیچ کس پاسخگو نیست
gigo.host
خرید نکنید دزدند

**Reply by Bahman5931** [reply]
نه اقا یک کم دیر جواب میدن اما دزد نیستن دمشون گرم سروری که گرفتم تا حالا فیلتر نشده
